pkg/minikube/cluster/local/local.go

```diff
@@ -0,0 +1,201 @@
+package local
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"path"
+	"strconv"
+	"time"
+
+	"github.com/docker/machine/libmachine/drivers"
+	"github.com/docker/machine/libmachine/engine"
+	"github.com/docker/machine/libmachine/log"
+	"github.com/docker/machine/libmachine/mcnflag"
+	"github.com/docker/machine/libmachine/mcnutils"
+	"github.com/docker/machine/libmachine/state"
+	"path/filepath"
+)
+
+type Driver struct {
+	*drivers.BaseDriver
+	EnginePort int
+	SSHKey     string
+}
+
+const (
+	defaultTimeout = 1 * time.Second
+)
+
+// GetCreateFlags registers the flags this driver adds to
+// "docker hosts create"
+func (d *Driver) GetCreateFlags() []mcnflag.Flag {
```

Comment:
We'll never actually go through these codepaths in minikube, so feel free to leave these functions blank (GetCreateFlags, SetConfigFromFlags).